src/geo/points.ts

```diff
@@ -72,7 +79,34 @@ const distance = (pointFrom: Point, pointTo: Point): number | null => {
   return earthRadius * c
 }
 
+const equals = (point1: Point, point2: Point): boolean => {
+  if (point1 === null && point2 === null) return true
+  if (point1 === null || point2 === null) return false
+  return point1.srs === point2.srs && point1.x === point2.x && point1.y === point2.y
+}
+
+const isValid = (point: Point): boolean => {
+  if (!point || !isFilled(point)) return false
+
+  try {
+    SRSs.getSRSByCode(point.srs)
+  } catch (e) {
+    // SRS not found
+    return false
+  }
+
+  const pointLatLong = toLatLong(point)
+  if (!pointLatLong || equals(pointLatLong, INVALID_LAT_LONG_POINT)) return false
+
+  return Numbers.between(pointLatLong.x, -180, 180) && Numbers.between(pointLatLong.y, -90, 90)
+}
+
+const toString = (point: Point): string => `SRID=${point.srs};POINT(${point.x} ${point.y})`
+
 export const Points = {
   parse,
   distance,
+  equals,
+  isValid,
+  toString,
 }
```

Comment:
better to move each function to its now folder

Reply:
do you mean it's own file? ok

Reply:
yeah :)